Author: Arthur de Jong <arthur@west.nl>
Description: fix a problem in the loc() function to not exit
--- a/chkrootkit
+++ b/chkrootkit
@@ -1178,7 +1178,7 @@
             .)
             if test -d $dir/$thing; then
                     echo $dir
-                    exit 0
+                    return 0
             fi
             ;;
             *)
@@ -1187,7 +1187,7 @@
             done
             if test -f $thisthing; then
                     echo $thisthing
-                    exit 0
+                    return 0
             fi
             ;;
             esac
@@ -1197,7 +1197,7 @@
     else
       echo "${ROOTDIR}${dflt}"
     fi
-    exit 1
+    return 1
 }

 getCMD() {
